pp/integrations/plugin-packs/procedures/network-ruckus-smartzone-snmp.md

```diff
@@ -111,92 +169,521 @@ To use this pack, the SNMP service must be properly configured on your Ruckus de
 
 Your centreon server must be able to reach the Ruckus device over UDP/161 SNMP port.
```

Comment:
use the standard sentence
also, in the previous sentence, it says "pack" instead of "connector"